code/__DEFINES/food.dm

```diff
@@ -177,6 +177,14 @@ DEFINE_BITFIELD(food_flags, list(
 
 #define BLACKBOX_LOG_FOOD_MADE(food) SSblackbox.record_feedback("tally", "food_made", 1, food)
 
+/// Point water boils at
+#define WATER_BOILING_POINT 373
+/// Point at which soups begin to burn at
+#define SOUP_BURN_TEMP 540
```

Comment:
this should also be reliant on `WATER_BOILING_POINT` somehow

Reply:
i still think it would be neat if we could incorporate boiling -> burning but whatever